apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -122,7 +122,12 @@ const Ticket = new GQLListSchema('Ticket', {
             ref: 'User',
             many: true,
         },
-
+        files: {
```

Comment:
It’s  looks  like heavy queries.  It’s better to filter Ticket files by ticket rather than get all files for ticket. Do you really need it??

Reply:
It's a standard keystone Relation one from the ticket and many from files. The only thing that is storing in the database is a "ticketId" field in ticket files table. Why do you think that it is heavy? And making this relation helps to get ticket together with its files.

Reply:
I remove files from ticket. No files are loaded on update ticket and view ticket through useobjects